test/components/resizable-bar.test.js

```diff
@@ -0,0 +1,63 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
+import '../../src/components/resizable-bar.js';
+
+// Mock PointerEvent if it's not defined in the testing environment
+if (typeof PointerEvent === 'undefined') {
+  global.PointerEvent = class extends MouseEvent {
+    constructor(type, params) {
+      super(type, params);
+    }
+  };
+}
```

Comment:
```suggestion
  global.PointerEvent = class extends MouseEvent { };
}
```
Should do the trick.

Reply:
Done!